integration_tests/run.sh

```diff
@@ -1,37 +1,56 @@
 #!/bin/bash
 # Script for jenkins to run tests on polar2grid.
 
-set -exf
+set -ex
 export PATH="/usr/local/texlive/2019/bin/x86_64-linux":$PATH
 cd "$WORKSPACE"
 # Activate conda for bash.
 /data/users/davidh/miniconda3/bin/conda init bash
 # Restart the shell to enable conda.
 source ~/.bashrc
-conda env update -n jenkins_p2g_env -f build_environment.yml --prune
+conda env update -n jenkins_p2g_env -f build_environment.yml
```

Comment:
Since you aren't pruning anymore (which I think I told you not to do because of performance issues), doesn't the environment always include all of the "jenkins_environment.yml" packages? Another possibly dumb option would be have one "build" environment (only build_environment.yml is used) and one "test" environment for running behave and building documentation. Not sure it is worth it but since you are using `conda env update` maybe it wouldn't be that bad after the first creation.

Reply:
Pruning is bugged and doesn't actually remove any packages not listed in the yaml file. I like the idea of having two environments. The extra time invested in to making two environments is relatively minuscule and it makes an impact on the size of the tarball.